src/components/OcSelect.vue

```diff
@@ -59,11 +75,40 @@ export default {
         return search.length ? fuse.search(search).map(({ item }) => item) : fuse.list
       },
     },
+    /**
+     * Disable the select component
+     */
     disabled: {
       type: Boolean,
       required: false,
       default: false,
     },
+    /**
+     * Label of the select component
+     * ATTENTION: this shadows the vue-select prop `label`. If you need access to that use `optionLabel`.
+     */
+    label: {
+      type: String,
+      default: null,
+    },
+    /**
+     * Key to use as label when option is an object
+     * NOTE: this maps to the vue-select prop `label`
+     */
+    optionLabel: {
+      type: String,
+      default: null,
+    },
+  },
+
+  computed: {
+    additionalAttributes() {
+      const additionalAttrs = {}
+      if (this.optionLabel) {
+        additionalAttrs["label"] = this.optionLabel
+      }
+      return { ...this.$attrs, ...additionalAttrs }
```

Comment:
`input-id` needs to be removed from `this.$attrs` because you already set it (with a different value) directly as a prop on the `vue-select`. Probably better to add `additionalAttrs["input-id"] = this.id` so that it overrules the input-id property from `this.$attrs` if it exists.

Reply:
Good catch!